experimental/integrations/prototype-spark/build.gradle

```diff
@@ -40,14 +42,14 @@ dependencies {
 }
 test {
   testLogging.showStandardStreams = true
-  jvmArgs '-javaagent:marquez-spark-agent/build/libs/marquez-spark-agent-all.jar'
+  jvmArgs '-javaagent:marquez-spark-agent/build/libs/marquez-spark-agent-all.jar=/api/v1/namespaces/ns_name/jobs/job_name/runs/ea445b5c-22eb-457a-8007-01c7c52b6e54'
```

Comment:
I wondering if we can make the agent argument a comma delimited string of key/value pairs (or possibly a configuration file)?

```
namespace=ns_name,job=job_name,run=ea445b5c-22eb-457a-8007-01c7c52b6e54
```